app/database.js

```diff
@@ -12,6 +12,19 @@
 define(['sqlite', 'app/config'], function (sqlite, config) {
 	// Check if SQL.js has been loaded through AMD
```

Comment:
If this still has to be done add some keyword so that you can find it easily, but even better, make a task on the backlog.